dmutils/email/dm_notify.py

```diff
@@ -82,32 +89,38 @@ def get_reference(email_address, template_id, personalisation=None):
         return hash_string(details_string)
 
     @staticmethod
-    def get_error_message(email_address, error):
+    def get_error_message(email_address, template_name, error):
```

Comment:
@samuelhwilliams has helpfully pointed out that this is a breaking change; but, as it isn't used anywhere outside of `DMNotifyClient`, I reckon it's probably fine to leave this change as a minor bump.